### DISCUSSION TITLE
Can't deploy my .net7 desktop app to github packages (403 forbidden)

### DISCUSSION THREAD

**Discussion by flarive** [community/community#63182] [Packages]
### Select Topic Area

Question

### Body

Hello,

I'm trying for several days to deploy a nuget package freshly created with github actions to github packages.
My nuget package is successfully created and uploaded to nuget.org but it fails when being deployed to github packages.

My repo : https://github.com/flarive/Neumorphism.Avalonia
My workflow publish script : https://github.com/flarive/Neumorphism.Avalonia/blob/main/.github/workflows/publish.yml

I have this error in job build result :

Run dotnet nuget push "neumorphism.avalonia.0.*.nupkg" --api-key *** --source "github"
Pushing Neumorphism.Avalonia.0.5.0.nupkg to 'https://nuget.pkg.github.com/eviral'...
  PUT https://nuget.pkg.github.com/eviral/
**warn : Your request could not be authenticated by the GitHub Packages service. Please ensure your access token is valid and has the appropriate scopes configured.**
  Forbidden https://nuget.pkg.github.com/eviral/ 155ms
**error: Response status code does not indicate success: 403 (Forbidden).**

Usage: dotnet nuget push [arguments] [options]
--------------------------------------------------------
Arguments:
  [root]  Specify the path to the package and your API key to push the package to the server.

Options:
  -h|--help                      Show help information
  --force-english-output         Forces the application to run using an invariant, English-based culture.
  -s|--source <source>           Package source (URL, UNC/folder path or package source name) to use. Defaults to DefaultPushSource if specified in NuGet.Config.
  -ss|--symbol-source <source>   Symbol server URL to use.
  -t|--timeout <timeout>         Timeout for pushing to a server in seconds. Defaults to 300 seconds ([5](https://github.com/flarive/Neumorphism.Avalonia/actions/runs/5796565317/job/15710385276#step:9:6) minutes).
  -k|--api-key <apiKey>          The API key for the server.
  -sk|--symbol-api-key <apiKey>  The API key for the symbol server.
  -d|--disable-buffering         Disable buffering when pushing to an HTTP(S) server to decrease memory usage.
  -n|--no-symbols                If a symbols package exists, it will not be pushed to a symbols server.
  --no-service-endpoint          Does not append "api/v2/package" to the source URL.
  --interactive                  Allow the command to block and require manual action for operations like authentication.
  --skip-duplicate               If a package and version already exists, skip it and continue with the next package in the push, if any.

Error: Process completed with exit code 1.
------------------------------------------------------------

I don't know why i have a 403 forbidden, i have tried to create a classic github token with repo+packages full rights, nothing works.

Thanks for your help

Eviral

**Comment by flarive** [comment]
Nobody ?
Do you need more info ?

**Comment by rainbow1016** [comment]
very good question!

**Comment by flarive** [comment]
It was a mistake in the github packages URL, the github support team helped me to find the reason.
Now it works.

**Reply by lostmsu** [reply]
What kind of mistake was it?

**Reply by flarive** [reply]
A bad deploy github packages url, i replaced my repo name with my user name or something like that
